site/content/integrate/marketplace-submissions/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Contribute to the Marketplace"
+heading: "Contribute to the Marketplace"
+description: "If you’ve built an integration, playbook, or tool which extends Mattermost functionality, consider sharing it with the community on the Mattermost Web Marketplace."
+weight: 100
+aliases: 
+  - /extend/plugins/community-plugin-marketplace/s
+---
+
+The {{<newtabref title="Mattermost Web Marketplace" href="https://mattermost.com/marketplace/">}} supports discovery, installation, and updates of extensions of the Mattermost platform. If you have built a plugin, app, playbook, integration, or tool that helps other users get more from Mattermost, the web Marketplace is a great way to get feedback on your contribution and help make it more popular. Once your submission is accepted to the Web Marketplace, Mattermost will also send you swag!
+
+All contributions are eligible following a review from our team. Apps and plugins that have been thoroughly reviewed by Mattermost will be eligible for our in-product marketplace as well with some additional requirements. Our in-product Marketplace enables customers to install and manage their apps and plugins directly within the product.
+
+## Types of integrations to submit to the web marketplace
```

Comment:
```suggestion
## Types of integrations to submit to the Web Marketplace
```

Reply:
@hanzei -We're aligning on "Mattermost Marketplace" as a formal product name that should be capitalized. Right now, our marketplace solution contains 2 parts: a web-based part and an in-product part. When the docs need to clarify which part, we'll leave those terms lowercase.

Long term, this issue will resolve itself with a single Mattermost Marketplace entity.